src/index.html

```diff
@@ -11,7 +11,206 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="PersonalInformation" id="PersonalInformation">
+        <legend>Personal information</legend>
+        <label class="form_field" id="L1">
+          <div>
+            Surname:
+            <input
+              name="Surname"
+              type="text"
+              autocomplete="off"
+              required
+              minlength="5"
+              maxlength="35"
+            >
+          </div>
+        </label>
+
+        <label class="form_field">
+          <div>
+            Name:
+            <input
+              name="Name"
+              type="text"
+              autocomplete="off"
+              required
+              minlength="5"
+              maxlength="25"
+            >
+          </div>
+        </label>
+
+        <label class="form_field">
+          <div>
+            How old are You?
+            <input
+              name="Age"
+              id="Age"
+              type="number"
+              required
+              min="1"
+              max="100"
+              value="12"
+            >
+          </div>
+        </label>
+
+        <label class="form_field">
+          <div>
+            Full date of birth:
+            <input
+              name="submissionDate"
+              type="date"
+              required
+            >
+          </div>
+        </label>
+
+        <label class="form_field" id="L2">
+          <div>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="Terms"
+              id="terms"
+              required
+            >
+          </div>
+        </label>
+      </fieldset>
+
+
+      <fieldset class="Registration" id="Registration">
+        <legend>Registration</legend>
+        <label class="form_field" id="L3">
+          <div>
+            E-mail:
+            <input
+              name="Email"
+              type="email"
+              required
+              minlength="10"
+              maxlength="35"
+              placeholder="email@example.com"
+            >
+          </div>
+        </label>
+
+        <label class="form_field" id="L4">
+          <div>
+            Password:
+            <input
+              name="Password"
+              type="password"
+              required
+              minlength="5"
+              maxlength="25"
+            >
+          </div>
+        </label>
+      </fieldset>
+
+      <fieldset class="AIFAY" id="AIFAY">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field" id="L5">
```

Comment:
It doesn't work correctly. Fix labels
![chrome-capture-2023-6-26](https://github.com/mate-academy/layout_html-form/assets/99403198/c019c00e-11a6-4d13-b743-9a90bb9025cb)